src/libs/ajax/User.ts

```diff
@@ -130,20 +130,14 @@ export const User = {
 
   addRoleToUser: async (userId: number, roleId: number): Promise<DuosUserResponse> => {
     const url = `${await getApiUrl()}/api/user/${userId}/${roleId}`
-    const res = await fetchAny(
-      url,
-      mergeAll([Config.authOpts(), { method: 'PUT' }]),
-    )
-    return res.json()
+    const res = await axios.put(url, null, Config.authOpts())
```

Comment:
See https://broadworkbench.atlassian.net/browse/DT-2086
Not sure we want to add axios here if we are going to move away from it